ThingIFSDK/ThingIFSDK/CountingTarget.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  CountingTarget.swift
+//  ThingIFSDK
+//
+//  Created on 2017/01/06.
+//  Copyright (c) 2017 Kii. All rights reserved.
+//
+
+import Foundation
+
+/**
+ A class to specify counting target.
+
+ This class contains data in order to count with
+`ThingIFAPI.count(_:_:_:clause:firmwareVersion:completionHandler:)`.
+ */
+open class CountingTarget: NSObject, NSCoding {
```

Comment:
@yongpingchen 

Umm, `CountingTarget` has `FieldType` enum. So If we change to `CountedField`, The enum becomes `CountedField.FieldType`. This is my concern.

What do you think?